File: Codeforces/Problemset/2009_C.py
import math

for _ in range(1, int(input()) + 1):
    x, y, k = map(int, input().split())
    
    jumps = 0
    
    if y > x - k:
        jumps = 2 * math.ceil(y / k)
    else:
        jumps = 2 * math.ceil(x / k) - 1
    
    print(jumps)

File: Codeforces/Problemset/2009_C.c
#include <stdio.h>
#include <math.h>

void solve()
{
    int x, y, k;

    scanf("%d %d %d", &x, &y, &k);

    x = ceil(x * 1.0 / k);
    y = ceil(y * 1.0 / k);

    if (y >= x) {
        printf("%d\n", 2 * y);
    }
    else {
        printf("%d\n", 2 * x - 1);
    }
}

int main()
{
    int t;

    scanf("%d", &t);

    while (t--) {
        solve();
    }

    return 0;
}